torch/functional.py

```diff
@@ -265,17 +265,19 @@ def einsum(equation, *operands):
 Einstein summation convention.
 
 Args:
-    equation (string): The equation is given in terms of lower case letters (indices) to be associated
-           with each dimension of the operands and result. The left hand side lists the operands
-           dimensions, separated by commas. There should be one index letter per tensor dimension.
-           The right hand side follows after `->` and gives the indices for the output.
-           If the `->` and right hand side are omitted, it implicitly defined as the alphabetically
-           sorted list of all indices appearing exactly once in the left hand side.
-           The indices not apprearing in the output are summed over after multiplying the operands
-           entries.
-           If an index appears several times for the same operand, a diagonal is taken.
-           Ellipses `...` represent a fixed number of dimensions. If the right hand side is inferred,
-           the ellipsis dimensions are at the beginning of the output.
+    equation (string): The subscripts for the Einstein summation specified as comma separated list of
```

Comment:
It's wild how much documentation was packed into this argument description, and how little is in the function description itself. Maybe we can move some of this there and improve the description with examples?